2-Authorization-I/1-call-graph/SPA/src/authConfig.js

```diff
@@ -58,7 +58,7 @@ export const msalConfig = {
  * https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes
  */
 export const loginRequest = {
-    scopes: ['User.Read']
+    scopes: ['User.Read', 'Contacts.Read'],
```

Comment:
@salman90 is dynamic consent not working in the sample?

Reply:
@derisen The popup window is not stable when closing. I pumped the MSAL versions to include the fix that the MSAL team worked on, and when calling `useMsalAuthentication` hook with Popup, I redirect to an empty page. 

It works fine for me now, but I am keeping the consent on login as a safeguard.